django_project/ford3/static/js/qualification_form.js

```diff
@@ -1,18 +1,63 @@
+let multipleSelectedLists = {};
+let multipleSelectHandler = (e) => {
```

Comment:
What does this function do?

Reply:
this is handler for multiple selection for this specific form